### PR TITLE
Implement updated external link UI per spec

### PR DESCRIPTION
**Pull request by joshlory**
I haven't changed the font metrics of the level — we'll need to do a broader pass across all text levels to normalize text.

Spec:
pasted image at 2016_06_16 06_18 pm

Before:
screen shot 2016-07-12 at 4 23 51 pm

After:
screen shot 2016-07-12 at 4 20 26 pm

### CODE REVIEW

**Comment by tanyaparker** [comment]
Can the external link icon be on the left and indented, almost like replacing a bullet point? I like it to the left since that's how it's presented in the course overview.

Do you also want to try putting the disclaimer way way (50px+) below the Next resource button? I can see what @breville meant by it calling maybe too much attention to the disclaimer more than the link itself?

File: dashboard/app/views/levels/_external_link.html.haml [hunk 1]
```diff
@@ -2,11 +2,17 @@
...
+      %i.fa.fa-external-link-square.external-link-icon>
 
-  %div{style: 'padding-top: 100px'}!= t('external_links.disclaimer_md', support_url: 'https://support.code.org')
+  .external-link-disclaimer
+    %h2= t('external_links.disclaimer')
+    %strong= t('external_links.leaving_domain')
+    %p!= t('external_links.disclaimer_text', support_url: 'https://support.code.org')
 
-  - if @script.try(:professional_learning_course?) && @script_level && @level.class == ExternalLink
+  - if @script_level
```
Comment (tanyaparker) [inline dashboard/app/views/levels/_external_link.html.haml:15]: Why don't we need these checks anymore?

Comment (joshlory) [inline dashboard/app/views/levels/_external_link.html.haml:15]: The "Next resource" button should show up in all courses, not just PLC (although I need to update this to just say "Continue").

**Comment by joshlory** [comment]
In markdown levels we've been putting the "external" icon at the end of the link — should these be updated to the bullet style as well?

screen shot 2016-07-12 at 4 57 17 pm

File: dashboard/app/views/levels/_external_link.html.haml (hunk 1 above)
Comment (tanyaparker) [inline dashboard/app/views/levels/_external_link.html.haml:15]: Gotcha

**Comment by tanyaparker** [comment]
Yeah, I think we should update it to be the left everywhere.

**Comment by tanyaparker** [comment]
Otherwise it gets lost, whereas an icon to the left (and indented) brings your eye to it.

**Comment by breville** [comment]
Interesting.  I do think there's external precedent for links that go outside of our own site having an icon to the right of the text, but if we want to put it on the left that should like good.

**Comment by joshlory** [comment]
After code review feedback:
screen shot 2016-07-13 at 5 41 31 pm

**Comment by tanyaparker** [comment]
LGTM

**Pull request merged**